provision/setup-vagrant

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
This is designed for running on the host machine, e.g. in setting up `support` again